editor.planx.uk/src/@planx/components/shared/Preview/MoreInfoSection.tsx

```diff
@@ -1,30 +1,16 @@
 import Box from "@mui/material/Box";
+import { styled } from "@mui/material/styles";
 import Typography from "@mui/material/Typography";
-import makeStyles from "@mui/styles/makeStyles";
 import React from "react";
-import { FONT_WEIGHT_SEMI_BOLD } from "theme";
 
-const infoSectionStyles = makeStyles((theme) => ({
-  root: {
-    "& + $root": {
-      paddingTop: theme.spacing(3),
-      borderTop: `1px solid ${theme.palette.text.primary}`,
-    },
+const Root = styled(Box)(({ theme }) => ({
+  "& img, & p": {
+    marginBottom: "1rem",
+    marginTop: 0,
   },
-  title: {
-    paddingBottom: theme.spacing(4),
-  },
-  content: {
-    "& img, & p": {
-      marginBottom: "1rem",
-      marginTop: 0,
-    },
-    "& strong": {
-      fontWeight: FONT_WEIGHT_SEMI_BOLD,
-    },
-    "& em": {
-      fontStyle: "italic",
-    },
```

Comment:
This was redundant - all `em` tags already have this fontStyle applied

https://browserdefaultstyles.com/#em